e2e/wdio/platform/tests/input.e2e-spec.ts

```diff
@@ -0,0 +1,158 @@
+import { webDriver } from '../../driver/wdio';
+import inputPContent from '../fixtures/appData/input-page-contents';
+import inputTestData from '../fixtures/testData/input';
+import { InputPo } from '../pages/input.po';
+
+describe('Input should ', function() {
+    const inputPage = new InputPo();
+
+    beforeAll(() => {
+        inputPage.open();
+    });
+
+    afterEach(() => {
+        webDriver.refreshPage();
+    });
+
+    it('have input without label', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.defaultInput);
+        webDriver.waitForDisplayed(inputPage.defaultInput);
+        expect(webDriver.isExists(inputPage.autocompleteInputLabel)).toBe(false);
+    });
+
+    it('be able to type something with keyboard', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.defaultInput);
+        webDriver.waitForDisplayed(inputPage.defaultInput);
+        webDriver.setValue(inputPage.defaultInput, inputTestData.text);
+
+        expect(webDriver.getValue(inputPage.defaultInput)).toBe(inputTestData.text);
+    });
+
+    it('have associated label element to describe its purpose', () => {
+        expect(webDriver.getTextArr(inputPage.inputsLabels, 0, -2)).toEqual(inputPContent.labelsArray);
+        expect(webDriver.getText(inputPage.inputsLabels, 7)).toContain(inputPContent.favoriteColor);
+    });
+
+    it('by default accept all kinds of input values – alphabet, numerical, special characters', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.defaultInput);
+        webDriver.waitForDisplayed(inputPage.defaultInput);
+        webDriver.setValue(inputPage.defaultInput, inputTestData.text);
+        webDriver.addValue(inputPage.defaultInput, inputTestData.number);
+        webDriver.addValue(inputPage.defaultInput, inputTestData.special_characters);
+
+        expect(webDriver.getValue(inputPage.defaultInput))
+            .toEqual(inputTestData.text + inputTestData.number + inputTestData.special_characters);
+    });
+
+    it('impose any filters on the kind of input values the component receives (text)', () => {
+        webDriver.waitElementToBePresentInDOM(inputPage.textInput);
+        webDriver.waitForDisplayed(inputPage.textInput);
+        webDriver.addValue(inputPage.textInput, inputTestData.number);
+        webDriver.addValue(inputPage.textInput, inputTestData.special_characters);
+        webDriver.addValue(inputPage.textInput, inputTestData.text);
+
+        expect(webDriver.getValue(inputPage.textInput))
+            .toEqual(inputTestData.number + inputTestData.special_characters + inputTestData.text); // ???
+    });
+    // TODO: it is not working the same for manual and automation.
```

Comment:
Did you already create an issue? If so, please comment the issue # here so we will be able to check it later on.